README_VERSION.md

```diff
@@ -0,0 +1,6 @@
+# v1.0.0
+
+### Fix
+
+Updated security group rule ingress to have for_each instead of count function for the input var.security_groups as source_security_group_id values.
+
```

Comment:
There are releases already made, ref to the versions from GitHub website, and then add this new version accordingly. And for format use this [README_VERSION.md](https://github.com/studiographene/terraform-aws-ecs-task-definition-service/blob/master/README_VERSION.md)